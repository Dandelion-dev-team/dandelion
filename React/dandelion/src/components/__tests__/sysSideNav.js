import React from "react"
import renderer from "react-test-renderer"
import SideNav from "../sysSideNav"

describe("Project Card", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SideNav/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})