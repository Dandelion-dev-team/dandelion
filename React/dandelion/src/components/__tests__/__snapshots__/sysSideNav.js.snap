// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Card renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="sidenav"
  >
    <nav>
      <div
        className="logo"
      >
        <a
          exact={true}
          href="/maintenance/sysadmin/sysadmin-dashboard"
        >
          <img
            src="test-file-stub"
          />
        </a>
      </div>
      <a
        href="/maintenance/sysadmin/auth-maintenance"
      >
        Authority
      </a>
      <a
        href="/maintenance/sysadmin/node-maintenance"
      >
        Node
      </a>
      <a
        href="/maintenance/sysadmin/quantity-maintenance"
      >
        Quantity
      </a>
      <a
        href="/maintenance/sysadmin/school-maintenance"
      >
        School
      </a>
      <a
        href="/maintenance/sysadmin/sensor-maintenance"
      >
        Sensor
      </a>
      <a
        href="/maintenance/sysadmin/superuser-maintenance"
      >
        Superuser
      </a>
      <a
        href="/maintenance/sysadmin/tag-maintenance"
      >
        Tag
      </a>
      <div
        className="btn-row"
      >
        <button
          className="button"
          onClick={[Function]}
        >
          Log Out
        </button>
      </div>
    </nav>
  </div>
</div>
`;
